app/components/exercises/ExerciseList.js

```diff
@@ -46,7 +78,7 @@ const ExerciseList = ({ user, exercises, deleteExercise }) => (
             <TableRowColumn style={{width:"10px", "paddingRight": "8em"}}>
```

Comment:
can you place this on the upper right corner of the table? that would be consistent with our other uses of a delete button.